internal/spokes/spokes.go

```diff
@@ -585,19 +603,24 @@ func (r *spokesReceivePack) readCommands(_ context.Context) ([]command, []string
 	return commands, shallowInfo, capabilities, nil
 }
 
-func (r *spokesReceivePack) dumpPushOptions(ctx context.Context) error {
+func (r *spokesReceivePack) dumpPushOptions(ctx context.Context) ([]string, error) {
```

Comment:
Would it make sense to return a count only, rather than the push options? Spokes-receive-pack doesn't use the options, and storing the options in a slice would let the client increase the memory use of spokes-receive-pack somewhat arbitrarily.

Reply:
Change it here 65bdcf392cd8ab5c5dd05ee4ecb1793c85f9ebf2